server/game/cards/13.5-TB/BlackwaterRush.js

```diff
@@ -0,0 +1,30 @@
+const DrawCard = require('../../drawcard');
+
+class BlackwaterRush extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card.controller === this.controller
+            },
+            title: 'Have character gain one power',

```

Comment:
`title` isn't needed on reactions / interrupts, because they don't appear in a menu for the card.

Reply:
fixed and thanks!